demisto_sdk/commands/common/tests/tools_test.py

```diff
@@ -2342,7 +2359,7 @@ def test_no_text_to_remove(self):
         prefix = "<>"
         suffix = "</>"
         text = "some text"
-        tag_parser = TagParser(prefix, suffix, remove_tag_text=False)
+        tag_parser = TagParser("XSOAR")
```

Comment:
why?

Reply:
This was just as a default value i will change it to FAKE_LABEL